File: Selenium/Activities/Activity6.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver import ActionChains, Keys
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())
with (webdriver.Firefox(service= service) as driver):
    driver.get("https://v1.training-support.net/selenium/input-events")
    print("Home Page title is :",driver.title)
    action = ActionChains(driver)
    action.send_keys("V").key_down(Keys.CONTROL).send_keys("A").send_keys("C").key_up(Keys.CONTROL).pause(5).perform()
    driver.close()

File: Selenium/Activities/Activity6.java
package activities;
import org.openqa.selenium.By;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.interactions.Actions;

public class Activity6 {
    public static void main(String[] args){
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/input-events");
        System.out.println("home Page Title :" + driver.getTitle());
        Actions action = new Actions(driver);
        action.sendKeys("V").pause(500).build().perform();
       action.keyDown(Keys.CONTROL).sendKeys("a").sendKeys("c").keyUp(Keys.CONTROL).build().
                perform();
        driver.close();

    }
}
